feature_showcase/11_Selection.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <script type="text/javascript" src="https://demo.microstrategy.com/MicroStrategyLibrary/javascript/embeddinglib.js"></script>
+    <!-- This is for the jsfiddle button. You don't need it in your application.  -->
+    <script type="text/javascript" src="js/jsfiddle.js"></script>    
+    
+  </head>
+  <body>
+    
+  <div>This example shows how to make selections on visualization programmatically. </div>
+
+  <table class="visualTable">
+    <tr>
+      <td><button onclick="updateVisuals()">Get Visualizations </button><select id="visualSelector"></select></td>
+    </tr>
+    <tr>
+      <td><button onclick="getElements()">Get Elements </button></td>
+    </tr>
+    <tr>
+        <td><div id="elementPanel"></div>
+          Action: 
+          <select id="actionSelector">
+            <option value="replace">replace</option>
+            <option value="add">add</option>
+            <option value="remove">remove</option>
+          </select>
+        </td>
+    </tr>
+    <tr><td><button onclick="applySelection()">Apply </button></td></td></tr>
+
+  </table>
+
+
+    <div id="mydossier"></div>
+    <script type="text/javascript">
+      
+      var url = "https://demo.microstrategy.com/MicroStrategyLibrary/app/B7CA92F04B9FAE8D941C3E9B7E0CD754/4FF2758511EAF7F3DBF40080EF359F4F"
+      
+      selectedAttribute = new Set()
+
+      document.addEventListener("DOMContentLoaded", function(){
+      
+        var container = document.getElementById("mydossier")
+        //This page has multiple visualizations
+        
+        microstrategy.dossier.create({
+            url: url,
+            enableResponsive: true,
+            placeholder: container,
+            containerHeight: '800px',
+            enableCollaboration: false,
+            dockedTOC: {
+                canClose: true,
+                dockChangeable: false,
+                isDocked:false
+            },
+            navigationBar: {
+                enabled: false           
+            }          
+        }).then((dossier) => {
+          d = dossier
+          d.registerEventHandler(microstrategy.dossier.EventType.ON_PAGE_SWITCHED, reset)
+          d.registerEventHandler(microstrategy.dossier.EventType.ON_PAGE_LOADED, reset)
+          d.registerEventHandler(microstrategy.dossier.EventType.ON_VIZ_ELEMENT_CHANGED,handleVizElementChanged)
+          d.registerEventHandler(microstrategy.dossier.EventType.ON_GRAPHICS_SELECTED,handleGraphicSelected)
+          
+        })
+      })
+
+      handleVizElementChanged = function(e){
+        //console.log("handleVizElementChanged", e)
+      }
+
+      handleGraphicSelected = function(e){
+        //console.log("handleGraphicSelected", e)
+      }
```

Comment:
Why are these two functions are doing nothing? Do we still need them? Maybe better comments are needed.